plugins/importexport/onix30/filter/MonographONIX30XmlFilter.inc.php

```diff
@@ -493,7 +494,7 @@ function createProductNode($doc, $submission, $publicationFormat) {
 		$collateralDetailNode = $doc->createElementNS($deployment->getNamespace(), 'CollateralDetail');
 		$productNode->appendChild($collateralDetailNode);
 
-		$abstract = strip_tags($publication->getLocalizedData('abstract'));
+		$abstract = strip_tags($publication->getLocalizedData('abstract', $pubLocale));
```

Comment:
Maybe to use getData instead of getLocalizedDate -- theoretically, it could be that there is no abstract in submission primary locale and that there is abstract in a locale other than the submission primary locale which would then be exported here.